### ISSUE TITLE
Larger simulation timesteps question

### ISSUE THREAD

**Issue by buddih09** [eclipse-sumo/sumo#14410]
Hi, 

I have an OD dataset with a 1-minute time resolution. each trip in the dataset takes longer than 1 minute and start-end times are given to the nearest minute. So, it makes sense for me to also set the simulation timestep in Sumo to 1 minute, but I'm worried whether this might cause inaccuracies in routing, because some in between edges might take less than 1 minute to traverse. 

Could you please help me understand Sumo behavior when the simulation step is large?

Regards
Buddi

**Comment by m-kro** [comment]
Please don't. The sumo time step is really not meant to be much larger than 1s. The behavioural models of road users are executed only at the time steps, meaning a vehicle would travel blindly quite far if you increase the time step largely. Although it is not exactly the same thing, you kind of increase the reaction time as well. So vehicles cross at red or break early...

Can you rephrase why this 1min is so important for you? I assume when generating the trips, you can define 1min intervals with separate OD data...

**Comment by m-kro** [comment]
See this [FAQ entry](https://sumo.dlr.de/docs/FAQ.html#how_fast_can_sumo_run) about performance as well... 10k vehicles may still be manageable depending on the time range (how many travel at the same time).
